model-optimizer/extensions/front/tf/non_max_suppression_ext.py

```diff
@@ -21,6 +21,17 @@
 from mo.front.extractor import FrontExtractorOp
 
 
+class NonMaxSuppressionV2Extractor(FrontExtractorOp):
+    op = 'NonMaxSuppressionV2'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attrs = {'sort_result_descending': 1, 'center_point_box': 0, 'output_type': np.int32}
```

Comment:
I didn't find `center_point_box`. May you mean `box_encoding`?

Reply:
Great catch about `center_point_box`! It should be `'box_encoding': 'corner'` instead.

Also, please, remove `center_point_box` attribute from the NMS operation constructor.

`np.int32` is used because TF NMS implementation produces tensor with indices of type `int32`: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/image_ops.cc#L972.

Reply:
@rkazants, thanks for noting this! Renamed, please review.
Code in extractors is identical but despite this this extractor is needed - it renames op name from `NonMaxSuppressionV2` to `NonMaxSuppression` so that I can find this node in replacer by the common name for all NMSs.